components/insight/SRC/org/openmicroscopy/shoola/env/data/views/calls/ResultsSaver.java

```diff
@@ -77,14 +78,20 @@
      * Create a temporary file
      *
      * @param img The image object to handle.
+     * @param fileName The name to use if any.
      * @return See above.
      */
-    private File createFile(ImagePlus img)
+    private File createFile(ImagePlus img, String fileName)
```

Comment:
Only question on this PR: is this okay to do on a stable line? Or should create a `private File createFile(ImagePlus img)` method for backwards-compatibilty?